spk/homeassistant/src/installer.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+
+# Package
+PACKAGE="homeassistant"
+DNAME="Home Assistant"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PYTHON_DIR="/usr/local/python3"
+PATH="${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+USER="homeassistant"
+GROUP="users"
+VIRTUALENV="${PYTHON_DIR}/bin/python3 -m virtualenv"			#was "${PYTHON_DIR}/bin/virtualenv-3.4"
+CONFIG_DIR="${INSTALL_DIR}/etc/"
+TMP_DIR="${SYNOPKG_PKGDEST}/../../@tmp"
+
+SERVICETOOL="/usr/syno/bin/servicetool"
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Create a Python virtualenv
+    ${VIRTUALENV} --system-site-packages ${INSTALL_DIR}/env > /dev/null
+
+    # Install the wheels
+    ${INSTALL_DIR}/env/bin/pip3.4 install --use-wheel --no-deps --force-reinstall -f ${INSTALL_DIR}/share/wheelhouse -r ${INSTALL_DIR}/share/wheelhouse/requirements.txt > /dev/null 2>&1
```

Comment:
If you made the virtualenv with 3.4 binary you should be able to use `pip` here.